app/src/main/java/com/sci/recipeandroid/feature/personalize/domain/repository/PersonalizeRepository.kt

```diff
@@ -0,0 +1,9 @@
+package com.sci.recipeandroid.feature.personalize.domain.repository
+
+import com.sci.recipeandroid.feature.personalize.data.datasource.PersonalizeRemoteDataSource
+import com.sci.recipeandroid.feature.personalize.domain.model.PersonalizeDataModel
+
+
+class PersonalizeRepository(private val remoteDatasource: PersonalizeRemoteDataSource) {
```

Comment:
please use dependency inversion

Reply:
Updated in e2ceaf1519d2ad526bf9d419752063a820c4ad13